pandas/core/internals/blocks.py

```diff
@@ -244,6 +244,8 @@ def make_block(self, values, placement=None) -> "Block":
         """
         if placement is None:
             placement = self.mgr_locs
+        if self.is_extension:
+            values = _block_shape(values, ndim=self.ndim)
```

Comment:
@jbrockmendel is this not a conditional check?

Reply:
this is a conditional check, but `self.is_extension` doesnt tell us anything about whether `values` is an EA.

Reply:
if we are calling .make_block from an EA block, don't we by definition *know* that we have extension block values?

Reply:
no.  e.g. if we call `pd.isna(frame)` that will end up with something like:

```
res_values = pd.isna(block.values)
return block.make_block(res_values)
```

which will be `ndarray[bool]`